flexget/plugins/input/filmweb_watchlist.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import unicode_literals, division, absolute_import
+
+import logging
+
+from flexget import plugin
+from flexget.event import event
+from flexget.utils.cached_input import cached
+from flexget.entry import Entry
+
+try:
+    from filmweb.filmweb import Filmweb as FilmwebAPI
+    from filmweb.items import LoggedUser
+    from filmweb.exceptions import RequestFailed
+except ImportError:
+    pass
+else:
+    @event('plugin.register')
+    def register_plugin():
+        plugin.register(FilmwebWatchlist, 'filmweb_watchlist', api_ver=2)
+
+
+log = logging.getLogger('filmweb_watchlist')
+
+
+class FilmwebWatchlist(object):
+    """"Creates an entry for each movie in your Filmweb list."""
+
+    schema = {
+        'type': 'object',
+        'properties': {
+            'login': {'type': 'string', 'description': 'Can be username or email address'},
+            'password': {'type': 'string'},
+            'min_star': {
+                'type': 'integer',
+                'default': 0,
+                'description': 'Items will be processed with at least this level of "How much I want to see"'
+            }
+        },
+        'additionalProperties': False,
+        'required': ['login', 'password'],
+    }
+
+    @cached('filmweb_watchlist', persist='2 hours')
+    def on_task_input(self, task, config):
+        log.verbose('Retrieving filmweb watch list for user: %s', config['login'])
+
+        fw = FilmwebAPI()
+        log.verbose('Logging as %s', config['login'])
+
+        try:
+            fw.login(str(config['login']), str(config['password']))
```

Comment:
No need to call `str` here

Reply:
It doesn't work without `str`.

Reply:
Did you test on both python 2 and 3? If it expects bytes it need to be handled differently

Reply:
It works in Python 3 without `str`.  What should I do then?

Reply:
Please paste the error you get without it on python 2

Reply:
```
2017-12-21 13:27 CRITICAL plugin        populate-movie-list Authentication request failed, reason (20, u'Incorrect JSON format')
```

Reply:
add`from builtins import *`
and then readd the `str`.

Does this work correctly for both cases?

Reply:
I added this line
```python
from builtins import *  # noqa pylint: disable=unused-import, redefined-builtin
```

and left this line
```python
fw.login(str(config['login']), str(config['password']))
```
untouched

Works both python 2 and 3.

Is this what you wanted to me to change?

Reply:
yep

Reply:
done